nlu/parser/src/main/java/edu/stanford/nlp/sempre/corenlp/FullNLPAnalyzer.java

```diff
@@ -37,6 +37,7 @@
     public static class Options {
         @Option(gloss = "What CoreNLP annotators to run")
         public List<String> annotators = Lists.newArrayList(
+            "tokenize",
```

Comment:
Do we really need tokenize twice?

Reply:
@josephbirkner @Waguramu ?
git is the witness i havent touched that file :P